main.py

```diff
@@ -1,10 +1,45 @@
-import init_django_orm  # noqa: F401
+import os
+import django
+from db.models import Race, Skill, Guild, Player
+import json
 
-from db.models import Race, Skill, Player, Guild
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "settings")
+django.setup()
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r", encoding="utf-8") as file:
+        players_data = json.load(file)
+
+    for nickname, player_data in players_data.items():
+        race, _ = Race.objects.get_or_create(
+            name=player_data["race"]["name"],
+            defaults={"description": player_data["race"]["description"]}
+        )
+
+        for skill_data in player_data["race"]["skills"]:
+            Skill.objects.get_or_create(
+                name=skill_data["name"],
+                bonus=skill_data["bonus"],
+                race=race
+            )
+
+        guild = None
+        if player_data["guild"]:
```

Comment:
Ensure that 'player_data["guild"]' is checked for null or missing values to prevent potential KeyError or TypeError. Consider using 'player_data.get("guild")' to safely access the guild data.